dos/src/main.rs

```diff
@@ -414,34 +418,43 @@ fn run_dos_transactions<T: 'static + BenchTpsClient + Send + Sync>(
 
     let mut transaction_generator = TransactionGenerator::new(transaction_params);
 
+    let connection_cache_stats = Arc::new(ConnectionCacheStats::default());
+    let udp_client = UdpTpuConnection::new(target, connection_cache_stats);
```

Comment:
Since the dos tool now uses the `ConnectionCache` and sets whether to use QUIC based on a command-line parameter, you can do the same thing here and use a `Connection`, ie:

```
let connection_cache = ConnectionCache::new(cmd_params.tpu_use_quic);
let connection = connection_cache.get_connection(target);
// and later...
connection.send_wire_transaction_batch(&data);
```